octodns/manager.py

```diff
@@ -265,9 +273,22 @@ def sync(self, eligible_zones=[], eligible_sources=[], eligible_targets=[],
         if eligible_zones:
             zones = [z for z in zones if z[0] in eligible_zones]
 
+        aliased_zones  = {}
         futures = []
         for zone_name, config in zones:
             self.log.info('sync:   zone=%s', zone_name)
+            if 'alias' in config:
+                source_zone = config['alias']
+                # Check that the source zone is defined.
+                if source_zone not in self.config['zones']:
+                    self.log.exception('Invalid alias zone')
```

Comment:
There isn't an exception here top log, you're getting ready to create and throw one, so this probably needs to be

```suggestion
                    self.log.error('Invalid alias zone {}, target {} does '
                                   'not exist'.format(zone_name, source_zone))
```

or something similar.

Reply:
Fixed.